nautobot/core/testing/filters.py

```diff
@@ -58,6 +59,14 @@ class FilterTestCase(BaseFilterTestCase):
         queryset = None
         filterset = None
 
+        # dictionary of attributes and lookup expression implemented in q filter
+        # meant to skip attributes that use icontains but cannot append a random string to attribute
```

Comment:
I'm not sure I understand what this comment means - can you perhaps clarify?

Would probably also be good to document that if you're using a `SearchFilter` you probably don't need to define this?